arch/none/host/src/arch_interrupt.c

```diff
@@ -14,77 +14,82 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+/*
```

Comment:
Do not change these files in arch/none/host, Can you add a new arch e.g. arch/none/host-scmi-server ?